transmart-data/test_data/keycloak/keycloak-test-resource.json

```diff
@@ -0,0 +1,1542 @@
+{
+  "id" : "test",
```

Comment:
what about having this realm settings as part of the keycloak libery test resources?

I will add this point to the following ticket's description https://jira.thehyve.nl/browse/TMT-261